text/2018-08-29-destroy-range.md

```diff
@@ -0,0 +1,50 @@
+# Summary
```

Comment:
# Add the `DestroyRange` Interface to TiKV

Reply:
Thanks for all your comments!
According to the template in this repository, we do not really need a main title in the document.

Reply:
@MyonKeminta That's OK.